java-frontend/src/main/java/org/sonar/java/se/checks/UnclosedResourcesCheck.java

```diff
@@ -116,34 +137,39 @@ private static boolean needsClosing(Type type) {
         return false;
       }
     }
+    for (String excludedType : excludedTypesList) {
```

Comment:
If the list is `static`, then you don't have to remove the static modifier everywhere. However, I would prefer you to use a dedicated getter lazily loading the configuration of excluded types.